credits.txt

```diff
@@ -323,12 +329,14 @@ contributed to Endless Sky:
   oo13
   opusforlife2
   Ornok
+  ovari
   OverYrPaygrade
   pakyinw
   PaulBlay
   PeacefulPotato
```

Comment:
```suggestion
```
https://github.com/endless-sky/endless-sky/pull/9740#pullrequestreview-2002387480

> I'd like the change to the credits to be separated from the other changes here. Either open a new PR for it or it can be handled when we update the credits and stuff for 0.10.7 next month.

Reply:
On consideration, I'd prefer to leave it in since these are entirely separate accounts, not an account name changing.